src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -117,6 +117,12 @@ public abstract class SFBaseSession {
   // Whether enable returning timestamp with timezone as data type
   private boolean enableReturnTimestampWithTimeZone = true;
 
+  // Server side value
+  private boolean jdbcEnablePutGet = true;
+
+  // Connection string setting
+  protected boolean enablePutGet = true;
```

Comment:
any reason for making this protected instead of private?

Reply:
Done.